js/image/shared/Webcam.svelte

```diff
@@ -88,15 +91,14 @@
 			media_recorder.stop();
 			let video_blob = new Blob(recorded_blobs, { type: mimeType });
 			let ReaderObj = new FileReader();
-			ReaderObj.onload = function (e): void {
+			ReaderObj.onload = async function (e): Promise<void> {
 				if (e.target) {
-					dispatch("capture", {
-						//@ts-ignore
-						data: e.target.result,
-						name: "sample." + mimeType.substring(6),
-						is_example: false,
-						is_file: false
-					});
+					let _video_blob = new File([video_blob], "video.mp4");
```

Comment:
previously we had the logic `"sample." + mimeType.substring(6)` to get the file extension. Now it's always an mp4. Is that a safe assumption?

Reply:
Good point, it can be `.webm` or `.mp4`. Fixed.